src/test/java/api/support/fakes/FakeOkapi.java

```diff
@@ -329,6 +332,19 @@ private void registerLibraryHours(Router router) {
       });
   }
 
+  private void registerConfiguration(Router router) {
```

Comment:
Is there a reason that the fake configuration is using this mechanism rather than the standard storage interface fake?

It is my understanding that the calendar calculate openings endpoint needs this because of the way the calculations are canned responses (and not dependent upon the specified opening times themselves).

That limitation meant that following the closed due date changes, it is now not possible to use the circulation API tests with the real module instances. This diminished a significant source of integration feedback. I'd prefer we didn't add to this unless it is definitely necessary.

Please let me know if this doesn't make sense and I can try to explain further.